src/styles/settings.scss

```diff
@@ -24,30 +24,36 @@ $marzipan: #F7DEA1;
 $color-facebook: #39579a;
 $color-twitter: #69ace0;
 
+// Project colours
+$space-background: #09141A;
+$brand-color-main: #18BAB4;
+$brand-color-secondary: #0F2B3B;
+$dark-text: #0A212E;
+$grey-text: #D8D8D8;
+$light-text: #FFFFFF;
```

Comment:
I noticed that couple of colours here are duplicated ($white and some others) so maybe instead of:
`$light-text: #FFFFFF;` we could initialize it like so: `$light-text: $white;`